test/test_distribution_generic.py

```diff
@@ -40,6 +40,12 @@ def __getattribute__(self, attr):
     FAKES = _fakes()
 
 
+if get_backend() == "jax":
+    _expanded_dist_path = "backend_dist.ExpandedDistribution"
+elif get_backend() == "torch":
+    _expanded_dist_path = "backend_dist.torch_distribution.ExpandedDistribution"
```

Comment:
Looks like you need to define a dummy value of `_expanded_dist_path` when `get_backend() == "numpy"` to get test collection to work, even though the tests themselves are all skipped.

Reply:
Thanks Eli!